src/mapogcsld.h

```diff
@@ -40,6 +40,8 @@ MS_DLL_EXPORT int msSLDApplySLDURL(mapObj *map, const char *szURL, int iLayer,
 MS_DLL_EXPORT int msSLDApplySLD(mapObj *map, const char *psSLDXML, int iLayer,
                                 const char *pszStyleLayerName,
                                 char **ppszLayerNames);
+MS_DLL_EXPORT int msSLDApplyFromFile(mapObj *map, layerObj *layer,
+                                     char *filename);
```

Comment:
Internal function only. Doesn't need to be exported:

```suggestion
int msSLDApplyFromFile(mapObj *map, layerObj *layer, const char *filename);
```